web/packages/teleport/src/AccessRequests/service.ts

```diff
@@ -78,8 +78,8 @@ export async function getDurationOptions(
     clusterId,
     roles,
     resources,
-    '',
-    true
+    'placeholder-reason',
+    true /* dryRun */
```

Comment:
instead of doing this, can't we on the backend `if dryrun` ignore this field?

Reply:
Good call, I'll look into that.

Reply:
Now I started to wonder if we are 100% sure we'll never add `--dry-run` flag to `tsh request create`